static/js/poll_widget.js

```diff
@@ -244,6 +256,12 @@ exports.activate = function (opts) {
     function submit_option() {
         var poll_comment_input = elem.find("input.poll-comment");
         var comment = poll_comment_input.val().trim();
+        var data = poll_data.get_widget_data().comments;
+
+        if(poll_data.is_comment_taken(data, comment))
```

Comment:
The whitespace in this PR is all off -- please run the linter and fix the issues it flags before opening a PR.